package.json

```diff
@@ -77,9 +77,9 @@
     "clean:build": "rimraf .cache .eslintcache coverage",
     "clean:exports": "lerna run clean:exports --parallel --stream",
     "generate": "yarn plop",
-    "lint": "node --max-old-space-size=4096 node_modules/.bin/eslint --ext js,jsx,ts,tsx --cache",
+    "lint": "node --max-old-space-size=4096 node_modules/.bin/eslint --ext js,jsx,ts,tsx --cache --cache-strategy content",
```

Comment:
Since Git doesn't track timestamps on files for modifications, the linter cache would not work. I've set this to check the file contents instead, which is slower, but will actually hit the cache.